src/php/api.php

```diff
@@ -31,7 +31,19 @@
         break;
 
       case 'GET':
-        $menu->viewOrders($db_conn);
+        if(isset($_GET["action"]))
+          {
+            switch($_GET["action"])
+            {
+              case "view_all_orders": $menu->viewAllOrders($db_conn); break;
```

Comment:
Later I'll change the data structure, now still testing so no child array for orders.